discord/bot.go

```diff
@@ -667,10 +675,22 @@ func (guild *GuildState) handleMessageCreate(s *discordgo.Session, m *discordgo.
 						}
 					}
 
+					guild.PrivateStateMsg.privateChannelID = privateChannelIDTemp;
```

Comment:
As per the comment above, I don't think this needs to be populated from a temp var, can't you populate from the persistent guild data directly here?

Reply:
Similar answer up above

Reply:
Going along with this issue, I previously had the trackingChannels put into a global space so I could have them set in the new command and used in the prompt command.

With 2 new changes I'm not sure how I should approach making it available to both commands.
1. I don't want to use global variables anymore, so perhaps I'd use the privatemessagestate?
2. The trackingChannels are generated in message_handlers instead of bots now.